### DISCUSSION TITLE
Create new file operation via the API

### DISCUSSION THREAD

**Discussion by thumDer** [outline/outline#5490] [Ideas]
Currently it is possible to query file operations, and also donwload them, but I couldn't find a way to create a new one. It would be great for us to make shceduled backups of our knowledge base.

**Comment by tommoor** [comment]
Hi – you can use the `collections.export_all` endpoint to make scheduled backups.

**Comment by thumDer** [comment]
I wasn't paying enough attention, thanks for the advice, will check it out!
